src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int sideA;
+    private int sideB;
+    private int heigth;
+
+    public IsoscelesTrapezoid(String color, int sideA, int sideB, int heigth) {
+        super(color);
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.heigth = heigth;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("The " + getColor() + " isosceles trapez has a sideA of "
+                + sideA + ", a sideB of " + sideB + ", a heigth of "
+                + heigth + " and an area of " + getArea());
+    }
+
+    @Override
+    public double getArea() {
+        return (sideA + sideB) * heigth * 2;
```

Comment:
formula is incorrect
![image](https://github.com/mate-academy/jv-oop-advanced/assets/73041091/60589cd2-90d9-4098-aaa9-cabad613c732)